archive/100-theorems-list/70_perfect_numbers.lean

```diff
@@ -0,0 +1,671 @@
+import data.nat.totient
+import number_theory.quadratic_reciprocity
+
+open_locale classical
+open_locale big_operators
+
+section definitions
+
+variable (n : ℕ)
+
+def divisors : finset ℕ :=
+finset.filter (λ x : ℕ, x ∣ n) (finset.Ico 1 (n + 1))
+
+def proper_divisors : finset ℕ :=
+finset.filter (λ x : ℕ, x ∣ n) (finset.Ico 1 n)
+
+lemma not_proper_self :
+¬ n ∈ proper_divisors n :=
```

Comment:
```suggestion
  ¬ n ∈ proper_divisors n :=
```
If the theorem statement wraps over more than one line, then the subsequent lines should be indented.

The proof then begins without indenting.